app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java

```diff
@@ -74,6 +77,8 @@ protected void onCreate(Bundle savedInstanceState) {
         mapFragment = (MapFragment) getSupportFragmentManager().findFragmentById(R.id.add_edit_map_fragment);
         socStnFragment = (SocialSituationFieldFragment) getSupportFragmentManager().findFragmentById(R.id.social_situation_field_fragment);
 
+        vibeEventDBPath = "Users/" + mAuth.getCurrentUser().getUid() + "/VibeEvents";
```

Comment:
At some point we should probably have a refactor issue to change string concatenations to `String.format` but this is preference for the most part

Reply:
Agree